app/services/analytics_events.rb

```diff
@@ -2899,8 +2898,8 @@ def idv_sdk_selfie_image_capture_opened(captureAttempts: nil, **extra)
   # @param [Integer] size size of image added in bytes
   # @param [String] source
   # @param [Integer] width width of image added in pixels
-  # User uploaded a selfie using the file picker
-  def idv_selfie_image_file_uploaded(
+  # rubocop:disable Naming/VariableName,Naming/MethodParameterName
+  def idv_selfie_image_added(
```

Comment:
since we're renaming this, can we tag it with `@identity.idp.previous_event_name` too?

Reply:
Oh interesting! I didn't realize that was a thing that was done.

These are selfie events so they haven't been in use beyond staging yet - do you still think that is needed? I'm not opposed to it, though I do think there's a tradeoff in that someone using this event might then also try to search for the previous name, with not much added value.

Reply:
if the events never made it to prod, then yeah no need to tag it like that